db/migrate/20250124154917_create_live_result_tables.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class CreateLiveResultTables < ActiveRecord::Migration[7.2]
+  def change
+    create_table :live_results do |t|
+      t.references :registration, null: false, index: true
+      t.references :round, null: false, index: true
+      t.references :entered_by, type: :integer, null: false, foreign_key: { to_table: :users }
+      t.datetime :entered_at, null: false
+      t.integer :ranking
+      t.integer :best, null: false
+      t.integer :average, null: false
+      t.string :single_record_tag, limit: 255
+      t.string :average_record_tag, limit: 255
+      t.boolean :advancing, default: false, null: false
+      t.boolean :advancing_questionable, default: false, null: false
+      t.timestamps
+    end
+
+    add_index :live_results, [:registration_id, :round_id], unique: true
```

Comment:
I think this can just be `t.index` inside of `create_table`, but feel free to prove me wrong